waveorder/cli/apply_inverse_models.py

```diff
@@ -127,13 +127,10 @@ def birefringence_and_phase(
 
     # [biref and phase, 2]
     if recon_dim == 2:
-        # Load phase transfer functions
-        absorption_transfer_function = torch.tensor(
-            transfer_function_dataset["absorption_transfer_function"][0, 0]
-        )
-        phase_transfer_function = torch.tensor(
-            transfer_function_dataset["phase_transfer_function"][0, 0]
-        )
+        # Load transfer functions
+        U = torch.tensor(transfer_function_dataset["singular_system_U"][0])
+        S = torch.tensor(transfer_function_dataset["singular_system_S"][0, 0])
+        Vh = torch.tensor(transfer_function_dataset["singular_system_Vh"][0])
```

Comment:
Use `torch.from_numpy` to ensure no extra mem_cpy?

Reply:
thanks!